lntemp/node/config.go

```diff
@@ -104,7 +104,12 @@ func (cfg BaseNodeConfig) RESTAddr() string {
 
 // DBDir returns the holding directory path of the graph database.
 func (cfg BaseNodeConfig) DBDir() string {
-	return filepath.Join(cfg.DataDir, "graph", cfg.NetParams.Name)
+	subdir := "graph"
+	if cfg.DbBackend == lntest.BackendSqlite {
+		subdir = filepath.Join("chain", "bitcoin")
```

Comment:
Would revert this if we go w/ the option to more directly reflect the existing DB file tree.